sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/eslint/EslintBridgeServerImpl.java

```
@@ -261,6 +268,20 @@ public void startServerLazily(SensorContext context) throws IOException {
       // required for SonarLint context to avoid restarting already failed server
       throw new ServerAlreadyFailedException();
     }
+    try {
+      var providedPort = nodeAlreadyRunningPort();
+      // if SONARJS_EXISTING_NODE_PROCESS_PORT is set, use existing node process
+      if (providedPort != 0) {
+        port = providedPort;
+        serverHasStarted();
+        LOG.info("Will use existing Node.js process in port " + port);
+      }
+    } catch (IllegalStateException e) {
```

we should not catch this, just let it fail